.github/workflows/kitchen.yml

```diff
@@ -112,13 +112,15 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v4
       - name: Install chef-test-kitchen-enterprise
+        env:
+          HAB_AUTH_TOKEN: ${{ secrets.HAB_AUTH_TOKEN }}
```

Comment:
we can't use Github Secrets as this breaks community forks and PR jobs fail for them as they don't have access to our secrets. This is the same reason why we had to hard code a chef-client free license into kitchen.yml files vs using github actions secrets.

Reply:
GITHUB_TOKEN only works as that's autogenerated for the specific GHA workflow action that is running and has limited access. Custom secrets such as this one needed to connect to public habitat builder to download the chef-test-kitchen-enterprise habitat package doesn't work with forks as that'd be a security issue for the secret.

Reply:
Ah indeed. Yeah, that's right - static secrets vs session secrets.